release/tag_repos.sh

```diff
@@ -85,25 +86,30 @@ tag_repos() {
 	info "Creating tag ${kata_version} in all repos"
 	for repo in "${repos[@]}"; do
 		git clone --quiet "https://github.com/${OWNER}/${repo}.git"
-		pushd "${repo}"
+		pushd "${repo}" >> /dev/null
 		git remote set-url --push origin "git@github.com:${OWNER}/${repo}.git"
 		git fetch origin --tags
 		if git rev-parse -q --verify "refs/tags/${kata_version}"; then
 			info "$repo already has tag "
 		else
-			info "Creating tag ${kata_version} for ${repo}"
-			git tag -a "${kata_version}" -s -m "${PROJECT} release ${kata_version}"
+			tag="$kata_version"
+			[[ "packaging" == "${repo}" ]] && tag="${tag}-kernel-config"
```

Comment:
Nit: the double-square brackets are only (?) required for magic pattern matching I think so I'd use single ones for clarity generally.